runtime/coreclr-bridge.m

```diff
@@ -164,6 +164,12 @@
 	xamarin_mono_object_release ((MonoObject **) mobj);
 }
 
+void
+xamarin_mono_object_release (MonoString **mobj)
+{
+	xamarin_mono_object_release ((MonoObject **) mobj);
```

Comment:
Are this things inlined?

Reply:
For non-.NET, they're completely removed (#define'd to empty strings). For .NET, they might be inlined. That said, inlining these methods is unlikely to have any measurable benefit, we're spending a lot of time doing other stuff.